adafruit_ads1x15/ads1x15.py

```diff
@@ -53,6 +53,16 @@
     16: 0x0A00,
 }
 
+# Pins
+PIN0 = 0
+"""Analog Pin 0"""
+PIN1 = 1
+"""Analog Pin 1"""
+PIN2 = 2
+"""Analog Pin 2"""
+PIN3 = 3
+"""Analog Pin 3"""
```

Comment:
If the names are going to be changed I would be in favor of using `A0`, `A1`, `A2`, and `A3` so that they match the values shown on the silkscreen which is how we typically name pins in the core: 

<img width="708" height="495" alt="Image" src="https://github.com/user-attachments/assets/2fc3a173-a1f5-472a-b069-140805c426c2" />